app/views/users/show/_guides.html.erb

```diff
@@ -20,10 +20,10 @@
               id: guide.crop.id }, :class => 'action show crop-name') %>
           </td>
           <td>
-            <%= link_to(icon('edit'), { :controller => 'guides', :action => 'edit', :id => guide.id}, :class => 'action show') %>
+            <%= link_to('<i class="fa fa-edit"></i> Edit'.html_safe, { :controller => 'guides', :action => 'edit', :id => guide.id}, :class => 'action show') %>
           </td>
           <td>
-            <%= link_to(icon('trash'), guide_path(guide), :method => 'delete', :class => 'action show', data: {confirm: "Are you sure?"}) %>
+            <%= link_to('<i class="fa fa-trash"></i> Delete'.html_safe, guide_path(guide), :method => 'delete', :class => 'action show', data: {confirm: "Are you sure?"}) %>
```

Comment:
Thanks @simonv3 - I'll make these changes and push up later today. I saw how translate was used elsewhere in the codebase but wasn't quite sure if I needed the other languages translation texts first, so that's why I did it as just an html string.